core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -257,6 +266,7 @@ protected void doStop() throws Exception {
           sslContextFactory.setProtocol(config.getString(RestConfig.SSL_PROVIDER_CONFIG));
         }
 
+        sslContextFactory.setRenegotiationAllowed(false);
```

Comment:
is there a particular reason we are changing this setting?

Reply:
no, this is an artifact of rebasing against `5.1.x` and moving it to `5.0.x`. 

https://github.com/confluentinc/rest-utils/blob/5.1.x/core/src/main/java/io/confluent/rest/Application.java#L283

i will remove.